### ISSUE TITLE
[nextgen-editor] Unable to add files / images since latest update

### ISSUE THREAD

**Issue by angusmcb** [getgrav/grav-premium-issues#151]
Probably related to #142 but since it is now closed I'm not sure comments there will ever be seen, and this seems like an important issue.

Since the latest update to the nextgen-editor I am completely unable to add images or files in any way. This includes using the drag-and-drop from the 'media' part of the page, using the button on the toolbar, or dragging and dropping from my computer. I have tried in both Firefox and Chrome on Windows.

**Comment by w00fz** [comment]
That’s very odd. Could you check if you are getting any JS errors?

**Comment by PalmTreeVI** [comment]
getting same, thought it was me and reinstall and started from scrath and got same result (unable to add image with Next Gen Editor)

**Comment by w00fz** [comment]
I might have introduced this issue with latest release, while fixing another issue. I’ll investigate this

**Comment by angusmcb** [comment]
I think there are actually two different problems, because when dragging & dropping or adding content that is already on the website no image appears. However, when dragging and dropping from the computer an image appears but it is saved directly into the markdown as data.

I get the following error in the console when trying to drag and drop an already uploaded file:

```
Uncaught (in promise) TypeError: this is undefined
    e utils.js:79
    tryCatch vendor.min.js:76352
    invoke vendor.min.js:76582
    defineIteratorMethods vendor.min.js:76407
    Babel 4
    M app.js:1
    R utils.js:79
    t imageupload.js:63
    tryCatch vendor.min.js:76352
    invoke vendor.min.js:76582
    defineIteratorMethods vendor.min.js:76407
    Babel 4
    init imageupload.js:55
    fire emittermixin.js:199
    g bubblingemittermixin.js:162
    fire bubblingemittermixin.js:92
    fire domeventobserver.js:96
    onDomEvent clipboardobserver.js:83
    observe domeventobserver.js:79
    fire emittermixin.js:199
    n emittermixin.js:247
utils.js:79:7
```

**Comment by w00fz** [comment]
Apologies for this @angusmcb.

I identified the issue and will resolve and release ASAP.

**Comment by w00fz** [comment]
This is now resolved with v1.1.6
